### PR TITLE
Move meta schema to simtools

### PR DESCRIPTION
**Pull request by GernotMaier**
This PR is connected to the [simtools PR 750](https://github.com/gammasim/simtools/pull/750) and should be reviewed and merged **after** #750.

- meta schema schemas/jsonschema.yml moved to [simtools/simtools/schemas/data.metaschema.yml](https://github.com/gammasim/simtools/blob/read_table_meta/simtools/schemas/data.metaschema.yml)
- update CI workflow to validate schema files to use this new metaschema file
- in all schema files (no other changes; so it is enough to review one of them):
   - new URL for base_schema pointing to above new metadata (that URL should work only after #750 in simtools has been merged)

### CODE REVIEW

File: .github/workflows/schema-validation.yml
Comment (GernotMaier) [inline .github/workflows/schema-validation.yml:1]: This will fail until #750 in simtools is merged (as the schema file is not there yet)

File: environment.yml
Comment (GernotMaier) [inline environment.yml:1]: Apologies - this change is unrelated (but consistent with what we have in simtools)

File: schemas/MST_mirror_2f_measurements.schema.yml
Comment (GernotMaier) [inline schemas/MST_mirror_2f_measurements.schema.yml:1]: this is the only of all files below you should review (same changes in all files)!

File: .github/workflows/schema-validation.yml
Comment (GernotMaier) [inline .github/workflows/schema-validation.yml:1]: - metaschema moved to simtools repository
- renamed application

**Review by orelgueta — APPROVED** [review]

**Pull request merged**
